socs/agents/ucsc_radiometer/agent.py

```diff
@@ -68,7 +68,11 @@ def acq(self, session, params=None):
         session.set_status('running')
 
         while self.take_data:
-            r = requests.get(self.url)
+            try:
+                r = requests.get(self.url)
+            except ValueError:
+                self.log.info(f'Query to data failed due to request error: {r.status_code}')
+                pass
```

Comment:
If a ValueError is raised, this will log a message _and then continue on to execute the code below_.  If "r" has a value, e.g. from the previous loop iteration, that value will be recorded in the data stream.  This is duplicate data.

I think intead of "pass" here, you want a `pm.sleep()` and `continue`.